cmake/Modules/LibAddPlugin.cmake

```diff
@@ -351,8 +351,6 @@ function (add_plugin PLUGIN_SHORT_NAME)
 	add_library (${PLUGIN_OBJS} OBJECT ${ARG_SOURCES})
 	add_dependencies (${PLUGIN_OBJS} kdberrors_generated)
 
-	add_dependencies(${PLUGIN_OBJS} readme_${PLUGIN_SHORT_NAME}.c)
```

Comment:
Are you sure this dep is not needed or did you remove it because it did not work?

I think the idea was to retrigger plugin compilation after `README.md` is changed.